code/__HELPERS/_lists.dm

```diff
@@ -25,6 +25,9 @@
 #define LAZYADDASSOC(L, K, V) if(!L) { L = list(); } L[K] += list(V);
 #define LAZYREMOVEASSOC(L, K, V) if(L) { if(L[K]) { L[K] -= V; if(!length(L[K])) L -= K; } if(!length(L)) L = null; }
 #define LAZYACCESSASSOC(L, I, K) L ? L[I] ? L[I][K] ? L[I][K] : null : null : null
+//Not to be used with actual lazy lists, as these don't null it on clearing.
+#define LAZYLISTDUPLICATE(L) (L ? L.Copy() : null )
+#define LAZYNULLLIST(L) L = null
```

Comment:
I meant to move to the bottom, under this comment, `LAZYCLEARLIST` and `LAZYCOPY`, which are not to be used with lazy lists.
You can also include `SANITIZE_LIST` and `reverseList`, both of which don't work for lazy lists.

Reply:
This should be fixed up now